app-text/sgml-common/sgml-common-0.6.3-r6.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils prefix
+
+DESCRIPTION="Base ISO character entities and utilities for SGML"
+HOMEPAGE="http://www.iso.org/standard/16387.html"
+#SRC_URI="mirror://kde/devel/docbook/SOURCES/${P}.tgz"
+SRC_URI="https://dev.gentoo.org/~floppym/dist/${PN}/${P}-gentoo.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
+
+src_prepare() {
+	# We use a hacked version of install-catalog that supports the ROOT
+	# variable, puts quotes around the CATALOG files, and can be prefixed.
+	cp "${FILESDIR}/${P}-install-catalog.in" "${S}/bin/install-catalog.in"
+
+	epatch "${FILESDIR}"/${P}-prefix.patch
```

Comment:
please use apply

Reply:
more about eapply and eapi6 https://devmanual.gentoo.org/ebuild-writing/eapi/ https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/

Reply:
Thanks for the pointers 😀
Done